pkg/sql2pgroll/create_table_test.go

```diff
@@ -44,6 +44,10 @@ func TestConvertCreateTableStatements(t *testing.T) {
 			sql:        "CREATE TABLE foo(a int CHECK (a > 0))",
 			expectedOp: expect.CreateTableOp10,
 		},
+		{
```

Comment:
Should we add a test for a non-constant default like `now()`?

Reply:
I just changed the testcase to use a `now()` as you suggest; I don't think we need two testcases here.